tests/test_humaninteractions.py

```diff
@@ -14,131 +14,321 @@
-				'created_at': 'some_point_in_time_and_space',
-				'data_types': {},
-				'details': {
-                    "some": "randon text"
-				},
-				'event_type': 'Test Human Interaction Workflow',
-				'event_meta': {},
-				'investigation_id': MOCK_INVESTIGATION_ID,
-				'status_': 'open',
-				'is_duplicate': False
-			},
-			'execution_id': MOCK_EXECUTION_ID
-		},
-		'State_Config': {
-			'Name': MOCK_STATE_NAME,
-			'Parameters': {}
-		}
-	}
-}
-
-MOCK_MESSAGE_RESPONSE_ENTRY = dict_to_item({
-    "await_token": TEST_SFN_CONTEXT['task_token'],
-    "execution_id": TEST_SFN_CONTEXT['sfn_context']['execution_id'],
-    "fulfilled": False,
-    "investigation_id": TEST_SFN_CONTEXT['sfn_context']['artifacts']['event']['investigation_id'],
-    "message": TEST_MESSAGE,
-    "receiver": TEST_SFN_CONTEXT['sfn_context']['State_Config']['Name'],
-},convert_root=False)
-
-
-MOCK_DB_CONTEXT = {
-    "datetime": "some_point_in_time_and_space",
-    "execution_id": "mock_execution_id",
-    "investigation_id": "mock_investigation_id",
-    "results": {
-        'artifacts': {
-            'event': {
-                'id': MOCK_EVENT_ID,
-                'created_at': 'some_point_in_time_and_space',
-                'data_types': {},
-                'details': {
-                    "some": "randon text"
+def test_init_human_interaction():
+    test_message = {
+        "greeting": "Hello, World"
+    }
+    sfn_item_metadata = mock_db_context_with_sfn()
+    sfn_context = sfn_item_metadata['sfn_context']
+    state_handler = StateHandler(sfn_context, MockLambdaContext(), mock_integration_handler)
+    message_id  = init_human_interaction(state_handler.context, test_message)
+    client = boto3.client('dynamodb')
+    init_message_response_entry = client.get_item(
+        TableName=os.environ['SOCLESS_MESSAGE_RESPONSE_TABLE'],
+        Key={'message_id': dict_to_item(message_id)})['Item']
+    
+    mock_message_response_entry = dict_to_item({
+        "await_token": sfn_context['task_token'],
+        "execution_id": sfn_context['sfn_context']['execution_id'],
+        "fulfilled": False,
+        "investigation_id": sfn_context['sfn_context']['artifacts']['event']['investigation_id'],
+        "message": test_message,
+        "receiver": sfn_context['sfn_context']['State_Config']['Name'],
+    },convert_root=False)
+    mock_message_response_entry['datetime'] = init_message_response_entry['datetime']
+    mock_message_response_entry['message_id'] = dict_to_item(message_id)
+    assert mock_message_response_entry == init_message_response_entry
+
+def test_init_human_interaction_with_invalid_execute_context_key():
+    bad_context = {'hello': 'world'}
+    test_message = {
+        "greeting": "Hello, World"
+    }
+    with pytest.raises(Exception):
+        message_id  = init_human_interaction(bad_context, test_message)
+
+def test_init_human_interaction_fails_on_bad_variable_type():
+    bad_context = {
+                'task_token': print,
```

Comment:
Consider using the Exception class here instead of print, and document why this test exists.